pytorch_toolkit/action_recognition_2/model_templates/gesture_recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,69 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction
+summary: ASL Recognition based on S3D-MobileNet-V3.
+annotation_format: CustomAction
+classes: []
+dependencies:
+- sha256: d55d4a208e2a0d379026665d6d5bf7c191b9cc4c52672733df1df878a6b3d7c6
+  size: 38499945
+  source: https://download.01.org/opencv/openvino_training_extensions/models/asl/s3d-mobilenetv3-large-statt-kinetics700.pth
+  destination: imagenet1000-kinetics700-snapshot.pth
+- sha256: 3bb05087cb405f83479a5ae83ec6a8ad49163047569e0309f78f81e9a337708c
+  size: 43195590
+  source: https://download.01.org/opencv/openvino_training_extensions/models/asl/s3d-mobilenetv3-large-statt-msasl1000.pth
+  destination: snapshot.pth
+- source: ../../../../../pytorch_toolkit/ote/tools/train.py
+  destination: train.py
+- source: ../../../../../pytorch_toolkit/ote/tools/eval.py
+  destination: eval.py
+- source: ../../../../../pytorch_toolkit/ote/tools/export.py
+  destination: export.py
+- source: ../../../../../pytorch_toolkit/ote/tools/quantize.py
```

Comment:
each script is copied twice

Reply:
Do you propose to use `symlink` instead to preserve documentation unchanged everywhere?

Reply:
We have at least three choices. Personally I would prefer 3. But it is can be discussed.
1. Keep initially broken symlink
2. Copy twice
3. Move `tools` somewhere outside of `pytorch_toolkit/ote`

Maybe some other variants exist

Reply:
What do you mean here `to preserve documentation unchanged everywhere?`?

Reply:
I mean that in documentation we use something like this: "Run the command `python eval.py ...`" which assumes file copying.

Reply:
Why not use `tools` from already copied `ote` that are in `packages/ote` or create `symlink` after copying `ote` (`train.py` -> `packages/ote/tools/train.py`)? So, we get rid of redundant dependencies in `template.yaml`.

Reply:
To summarize, if don't want to change the logic of `instantiate_template.py` (due to possibly different implementations on other side) the only solution is to allow copy twice (anyway no future changes is expected). Alternatively, we can add some flag (aka `symlink: True`) into template.yaml and change the `instantiate.py` script behaviour for creating symlinks after copying `packages/ote` (as proposed by @DmitriySidnev). In favour the restrictions on `template.yaml` changes the last option is improbable.

Reply:
May be move this discussion into separate PR and preserve `copy twice` for a while?

Reply:
:+1: